commerce_coordinator/settings/base.py

```diff
@@ -409,13 +407,27 @@ def root(*path_fragments):
         ]
     },
     "org.edx.coordinator.titan.payment.superseded.v1": {
-        "fail_silently": False,  # TODO: Coordinator filters should NEVER be allowed to fail silently
         "pipeline": [
             'commerce_coordinator.apps.stripe.pipeline.UpdateStripePayment',
         ]
     }
 }
 
+
+def _set_fail_silently(inner_dict):
```

Comment:
Super appreciate you taking this on. A worry: settings are designed to be overridden from other settings files or by settings in the deployment environment. `_set_fail_silently()` will not run for the overrides.

What about creating a new `CoordinatorFilter(OpenEdxPublicFilter)` class that overrides [`get_filter_config`](https://github.com/openedx/openedx-filters/blob/b02caa997e80c82a15bca9df182c480d89b612c8/openedx_filters/tooling.py#L122C9-L122C26), and using `CoordinatorFilter` instead of `OpenEdxPublicFilter` in commerce-coordinator?

The override could, e.g. force all `fail_silently` to be `True`, and emit appropriate warning/error messages.

Reply:
ok fair enough, ill cut it out

Reply:
im marking this resolved as i have the changes in my index, they will come in together with a couple others.